db/schema.sql

```diff
@@ -0,0 +1,31 @@
+-- Tenants table (with the following attributes):
+-- id
+-- name
+-- age
+-- gender
+-- apartment_id
+--
+-- Apartments table (with the following attributes):
+-- id
+-- address
+-- monthly_rent
+-- sqft
+-- num_beds
+-- num_baths
+
+CREATE TABLE tenants (
+  tenants_id_seq SERIAL NOT NULL PRIMARY KEY,
+  name VARCHAR NOT NULL,
+  age INT,
+  gender VARCHAR,
+  apartment_id INT NOT NULL
```

Comment:
Keep in mind that if you set `apartment_id` to `NOT NULL` then a tenant must always be attached to an apartment. Nothing wrong with that, but know that you can't have any "floating" tenants now.